W-10877309 Exchange create custom policy for Flex Gateway jh (#188)
mulesoft/docs-exchange: modules/ROOT/pages/add-policy-implementation-files-to-published-policy-definition.adoc

> = Add Policy Implementation Files to Published Policy Definition

Reviewer: ```suggestion
= Add Policy Implementation Files to the Published Policy Definition
```

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> When a policy definition asset is published in Exchange, the asset details page displays a *Pending* flag next to *Implementations* in the left navigation bar. The flag indicates that the policy implementation has not been added. Add the policy implementation files to to the policy definition to enable the policy to be available in runtime. The required implementation files are a binary file such as a WebAssembly (WASM) file for Flex Gateway or JAR for Mule 4 runtime and a YAML metadata file. 

Reviewer: ```suggestion
When a policy definition asset is published in Exchange, the asset details page displays a *Pending* flag next to *Implementations* in the navigation bar. The flag indicates that the policy implementation has not been added. Add the policy implementation files to the policy definition to make the policy available in runtime. The required implementation files are a binary file such as a WebAssembly (WASM) file for Flex Gateway or JAR file for Mule, and a YAML metadata file. 
```

Reviewer: Do you need this sentence? I'm not sure it adds anything to the meaning of the test.

The flag indicates that the policy implementation has not been added.

Before adding the policy implementation files, ensure that the policy definition asset is in the Stable state.

To add the policy implementation files:

. In Exchange, select the policy asset to open the asset details page.
. Click *Implementations* in the left navigation bar.
. Click *Add implementation*.
. Enter the name for the implementation.
. Click *Choose file* to select the JAR or WebAssembly file.
. Click *Choose file* to select the YAML file.
. To edit the GroupId, AssetId, Version, and API version, click *Advanced*. 
+

> Exchange generates the group ID, asset ID, and version. You can change these values. You can also change an API asset’s version and API version separately. The advanced settings are most often used to change the asset version.

Reviewer 2: there is no API version in the implementations
```suggestion
Exchange generates the group ID, asset ID, and version. The advanced settings are most often used to change the asset version.
```

. Click *Add implementation*.
+

> The implementation displays in the *Manage implementations* table. From this table, manage the implementation versions of a policy asset. See xref:manage-versions.adoc[Manage Versions] for more information.

Reviewer: "...you can manage... (reads better)

Reviewer: Do you need this sentence?

You can also change an API asset’s version and API version separately.
